src/utils/hasName.js

```diff
@@ -0,0 +1,5 @@
+const _hasName = baladyiaName => {
```

Comment:
Please rename it to `hasName` here and in the places where you import it. Also rename `baladyiaName` to something like `givenName`